### ISSUE TITLE
MRAE performance

### ISSUE THREAD

**Issue by Fern-Luiz** [Deep-imagelab/AWAN#3]
Hello, while the paper shows  MRAE 0.06+ on Real world NTIRE 2020,  I use AWAN+ algorithm to test the validation set (provided by competition in google drive)on real word but get MRAE 0.1+. Do you have any different tricks that not published in your paper?

**Comment by Deep-imagelab** [comment]
You can download the pretrain models from the link- https://drive.google.com/drive/folders/1cn3q33tTgPaaAlYiGQ20fycxgo9m_F4s?usp=sharing

**Comment by Fern-Luiz** [comment]
> You can download the pretrain models from the link- https://drive.google.com/drive/folders/1cn3q33tTgPaaAlYiGQ20fycxgo9m_F4s?usp=sharing

I do used the pretrain models from link you provided, but  got MRAE 0.1+. Can you show the final validation result (.mat files)from AWAN+ of your team?

**Comment by Deep-imagelab** [comment]
> > You can download the pretrain models from the link- https://drive.google.com/drive/folders/1cn3q33tTgPaaAlYiGQ20fycxgo9m_F4s?usp=sharing
> 
> I do used the pretrain models from link you provided, but got MRAE 0.1+. Can you show the final validation result (.mat files)from AWAN+ of your team?

We have corrected this link.You can try again.The final verification set result is being uploaded to Google Drive, the link will be shared with you later.

**Comment by Fern-Luiz** [comment]
Thank you, look forword to your link!

**Comment by Deep-imagelab** [comment]
> Thank you, look forword to your link!

We have uploaded the corresponding results, please check carefully the README.md in the direction "AWAN_RealWorld\test\" .
